lib/mindwendel_web/templates/layout/root.html.heex

```diff
@@ -7,17 +7,12 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
     />
-    <%= csrf_meta_tag() %>
+    <meta name="csrf-token" content={get_csrf_token()} />
```

Comment:
added this one back here similar to the static.html.eex for now, I guess you shouldn't need it in a liveview but app.js was crashing on:

```
let csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute("content")
```

so none of the "live" features worked so I added it back in for now